orc8r/cloud/go/services/bootstrapper/bootstrapper/main.go

```diff
@@ -92,9 +94,15 @@ func createRegistrationServicers(srv *service.OrchestratorService) (protos.Cloud
 		glog.Fatalf("failed to get rootCA: %+v", err)
 	}
 
+	domainName, err := getDomainName()
+	if err != nil {
+		glog.Fatalf("failed to get domainName: %+v", err)
```

Comment:
Let's just make this an error. Because on-prem deployments may not have this var set, and it's not a show-stopping error